Negative correlation in event correlation (EQL) rules (#3709)
elastic/security-docs: docs/detections/rules-ui-create.asciidoc

[[create-eql-rule]]
==== Create an event correlation rule
. To create an event correlation rule using EQL, select *Event Correlation*, then:
.. Define which {es} indices or data view the rule searches when querying for events.
.. Write an {ref}/eql-syntax.html[EQL query] that searches for matching events or a series of matching events.
+

> TIP: When searching for a series of matching events, use the {ref}/eql-syntax.html#eql-missing-events[missing events] syntax to specify events that should _not_ happen.  

Reviewer: Maybe a different word at the end? It's unclear what it means if an event "should not happen." Does the syntax exclude events from the search?
```suggestion
TIP: When searching for a series of matching events, use the {ref}/eql-syntax.html#eql-missing-events[missing events] syntax to specify events that should _not_ be matched.  
```

Author: @joepeeples if I'm understanding the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/eql-syntax.html#eql-missing-events) correctly, negative correlation syntax should return events that don't match the conditions you specify. So it'd be like saying, "Find me all the events that _don't match_ this set of criteria." @paulewing @yctercero am I understanding this correctly? 

If I am, how does the following look:

```suggestion
TIP: To find events that don't match your specified conditions, use the {ref}/eql-syntax.html#eql-missing-events[missing events] syntax.  
```

Reviewer 2: @nastasha-solomon I like this second option better. 👍

Author: Revising this based on @paulewing's feedback on Slack:

 ```suggestion
TIP: To find events that are missing in a sequence, use the {ref}/eql-syntax.html#eql-missing-events[missing events] syntax.  
```
